platforms/ios/lib/WysiwygComposer/Sources/HTMLParser/Extensions/NSAttributedString+Range.swift

```diff
@@ -95,9 +109,17 @@ extension NSAttributedString {
                 .outOfBoundsAttributedIndex(index: attributedIndex)
         }
 
+        let discardableTextRanges = discardableTextRanges()
+        var actualIndex = attributedIndex
+
+        for discardableTextRange in discardableTextRanges {
+            if discardableTextRange.upperBound <= attributedIndex {
+                actualIndex -= discardableTextRange.length
+            }
+        }
```

Comment:
Think this raises a Swiftlint warning, I'm guessing we can e.g. write something like:

```swift
actualIndex -= discardableTextRanges
    .filter { $0.upperBound <= attributedIndex }
    .reduce(0) { $1 + $2.length }
```